chart_tests/test_git_sync_scheduler.py

```diff
@@ -0,0 +1,194 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import sys
+import unittest
+
+import yaml
+from kubernetes.client import models as k8s
+
+from .helm_template_generator import render_chart, render_k8s_object
+
+OBJECT_COUNT_IN_BASIC_DEPLOYMENT = 22
+
+
+class TestGitSyncScheduler(unittest.TestCase):
+    def setUp(self) -> None:
+        with open(
+            sys.path[0] + "/chart_tests/test_templates/" "git_sync_scheduler/git-sync-scheduler.yaml"
+        ) as file:
+            self.basic_object = yaml.load(file)
+        with open(
+            sys.path[0] + "/chart_tests/test_templates/" "git_sync_scheduler/git-sync-container-spec.yaml"
+        ) as file:
+            self.object_with_git_container_spec = yaml.load(file)
+        with open(
+            sys.path[0] + "/chart_tests/test_templates/" "git_sync_scheduler/git-sync-ssh-params.yaml"
+        ) as file:
+            self.object_with_ssh_params = yaml.load(file)
+        with open(
+            sys.path[0] + "/chart_tests/test_templates/" "git_sync_scheduler/git-sync-username.yaml"
+        ) as file:
+            self.object_with_username = yaml.load(file)
+        with open(
+            sys.path[0] + "/chart_tests/test_templates/git_sync_scheduler/" "git-sync-existing-claim.yaml"
+        ) as file:
+            self.object_with_existing_claim = yaml.load(file)
```

Comment:
This looks bad, either use python dicts directly, or create a function that takes a path, and return loaded_yaml